nextcord/client.py

```diff
@@ -1862,35 +1863,35 @@ def add_view(self, view: View, *, message_id: Optional[int] = None) -> None:
         Parameters
         ----------
         view: :class:`nextcord.ui.View`
-            The view to register for dispatching.
+            The view to register for dispatching or add to storage.
```

Comment:
This sounds weird

Reply:
I thought it sounded right because of the "to" at the beginning of the sentence. I'll add an extra one.